README.md

```diff
@@ -92,41 +90,41 @@ The supported `boot/iso/` sub-directories (in alphabetical order) are :
 
 [//]: # (distro-list-end)
 
-Any unpopulated directory will have the matching boot menu entry automatically
-disabled, so to skip any distribution, just don't copy any files into it.
+Unpopulated directories turns its matching boot menu entry off,
+so skip any distribution by not copying any files into it.
 
-Download the right ISO image(s) to the matching directory. If you require
-boot parameter tweaks, edit the appropriate `boot/grub2/inc-*.cfg` file.
+Download the right ISO image(s) to the matching directory.
+If you require boot parameter tweaks, edit the appropriate
+`boot/grub2/inc-*.cfg` file.
 
-Items order in the menu
-------------
+Item order in the menu
+----------------------
 
-Menu items for a distro are ordered by modification time of the iso files
-starting from the most recent ones. If some iso files have the same mtime, their
-menu items are ordered alphabetically.
+Distros as menu items are ordered by the modification time of their ISO files,
+starting with the most recent. If some ISO files have the same mtime,
+their menu items are ordered A–Z.
 
-Here is a generic idea how to keep it nicely ordered when you have multiple
+Here is a generic idea of how to keep it nicely ordered with multiple
 releases of some distro:
 
-- touch your **release** iso files with the release date
```

Comment:
I think touch lowercase was to hint at using that command, so maybe `touch` instead to make it clearer?